troubleshoot/ingest/opentelemetry/edot-collector/index.md

```diff
@@ -0,0 +1,30 @@
+---
+navigation_title: EDOT Collector
+description: Troubleshooting common issues with the EDOT Collector.
+applies_to:
+  stack:
```

Comment:
Q: Why does these pages also apply to stack?

Reply:
EDOT can be used with Stack deployments.

Reply:
Ahh, make sense.